src/classes/USER_FrozenInActiveUser_TDTM_TEST.cls

```diff
@@ -0,0 +1,159 @@
+/*
+    Copyright (c) 2017, Salesforce.org
+    All rights reserved.
+    
+    Redistribution and use in source and binary forms, with or without
+    modification, are permitted provided that the following conditions are met:
+    
+    * Redistributions of source code must retain the above copyright
+      notice, this list of conditions and the following disclaimer.
+    * Redistributions in binary form must reproduce the above copyright
+      notice, this list of conditions and the following disclaimer in the
+      documentation and/or other materials provided with the distribution.
+    * Neither the name of Salesforce.org nor the names of
+      its contributors may be used to endorse or promote products derived
+      from this software without specific prior written permission.
+
+    THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+    "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT 
+    LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS 
+    FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE 
+    COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, 
+    INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, 
+    BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; 
+    LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER 
+    CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT 
+    LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN 
+    ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE 
+    POSSIBILITY OF SUCH DAMAGE.
+*/
+
+/**
+* @author Salesforce.org
+* @date 2017
+* @group User
+* @description Trigger handler on User object to handle User status changes.
+*/
+
+@isTest
+public with sharing class USER_FrozenInActiveUser_TDTM_TEST {
+	//================================
+	// Constants
+	//================================
+	public static final String SYSTEM_ADMINISTRATOR = 'System Administrator';
+	public static final String ERROR_NOTIFICATION_RECIPIENT_ALL_SYS_ADMINS = 'All Sys Admins';
+	
+	
+	/*********************************************************************************************************
+    * @description Tests the handling of the TDTM_Runnable.Action for BeforeInsert 
+    * @return void
+    **********************************************************************************************************/
+    private static testMethod void testRunOverrideBeforeInser() {
```

Comment:
while I wouldn't have created the empty stubs in USER_FrozenInActiveUser_TDTM for all the trigger events you weren't handling, I was fine leaving them in.  But writing tests for them which can't actually assert anything seems misleading.  What benefit are these?  They seem a detriment, to me, because they imply that tests are written for the other trigger events, but in fact there is no testing logic here, and there is no valid way to write tests for functionality that doesn't yet exist or has even been defined.  I'd prefer seeing these removed.  thanks.

Reply:
I removed the tests for the empty stubs. 

Also, I'm open to removing the empty stubs. I included them as part of the design structure. To keep the structure we can replace the stubs with comments. Thoughts?